tests/components/nintendo_parental/test_config_flow.py

```diff
@@ -0,0 +1,152 @@
+"""Test the Nintendo Switch Parental Controls config flow."""
+
+from homeassistant import config_entries
+from homeassistant.components.nintendo_parental.const import CONF_SESSION_TOKEN, DOMAIN
+from homeassistant.const import CONF_API_TOKEN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from tests.common import MockConfigEntry
+
+mock_config_entry = MockConfigEntry(
+    domain=DOMAIN,
+    data={CONF_SESSION_TOKEN: "valid_token"},
+    unique_id="aabbccddee112233",
+)
+
+
+async def test_full_flow(hass: HomeAssistant, mock_authenticator_client) -> None:
```

Comment:
typing. Same for the other tests.
```suggestion
async def test_full_flow(hass: HomeAssistant, mock_authenticator_client: MagicMock) -> None:
```